generators/client/templates/react/src/test/javascript/spec/app/shared/auth/private-route.spec.tsx.ejs

```diff
@@ -25,11 +25,7 @@ describe('private-route component', () => {
     const error = comp.find('div.insufficient-authority');
     expect(error.length).to.equal(1);
     expect(error.find('.alert-danger').html()).to.equal(
-      <%_ if (enableTranslation) { _%>
-      '<div class="alert alert-danger"><span>translation-not-found[error.http.403]</span></div>'
-      <%_ } else { _%>
       '<div class="alert alert-danger">You are not authorized to access this page.</div>'
-      <%_ } _%>
```

Comment:
For whatever reason, my tests was able to load translation files with i18n enabled. Gonna wait to see if Travis has the same result.